src/components/breadcrumb/breadcrumb.html

```diff
@@ -1,4 +1,4 @@
-<div class="bx--breadcrumb bx--breadcrumb--no-trailing-slash">
+<nav class="bx--breadcrumb bx--breadcrumb--no-trailing-slash" aria-label="breadcrumb">
```

Comment:
Nvm, as soon as I posted the question I found helpful docs from W3 that says this should be okay: 

![image](https://user-images.githubusercontent.com/4185382/29132840-26c073f2-7cf7-
11e7-9230-7657c7b7e7fe.png)

When there's multiple navigation groups, when the screen reader enters the breadcrumb group, the user will know to distinguish the breadcrumb nav from other navs based on the aria-label. I'm good with this PR now 👍 ✅ lol

Source: https://www.w3.org/WAI/GL/wiki/Grouping_related_links_using_the_nav_element